### DISCUSSION TITLE
@DIGITALCRIMINAL      We Need A Video Tutorial Please

### DISCUSSION THREAD

**Discussion by Corpse405** [UltimaHoarder/UltimaScraper#1365] [Ideas]
Trying to follow the tutorial steps is pretty confusing being a non-coder/ someone who doesn't know any code. There used to be a video on YT how to do it and it was amazing the one time I got it to work, but the video has since been taken down. 

We really need a video tutorial or maybe a WikiHow style with pictures for all of us who are more visual learners. Hell I'm a video editor by trade I'll make a video tutorial **for free** If someone can show me how to do it.

**Comment by UltimaHoarder** [comment]
I dunno about a video, but I've fixed certain parts of the script that was crashing for fresh downloaders and I've changed the authentication process. It's much easier now.

**Reply by jalh1** [reply]
unfortunately I am inept, but I want to install the DCOF program that you will release. It doesn't matter if I have to pay you, just tell me and we will arrange the price and payment method. I remain at your service. I also need to know how to use it

**Comment by Luke-L** [comment]
there was one a few months ago but it got removed. You might be able to find a way in, but idk 
https://web.archive.org/web/20210317042445if_/https://www.youtube.com/watch?v=6Fy1Wy9Fpv8

**Comment by LitoStarr** [comment]
Why would this need a video tutorial? The instructions are detailed enough.

**Reply by UltimaHoarder** [reply]
I promise you, People are BRAINDEAD and can't follow simple instructions.

**Reply by DrEvilHomer** [reply]
Apparently I am just braindead then.  As someone that has never used or messed with this type of thing it makes literally zero sense.  Something visual or associative would be great.

**Reply by jalh1** [reply]
unfortunately I am inept, but I want to install the DCOF program that you will release. It doesn't matter if I have to pay you, just tell me and we will arrange the price and payment method. I remain at your service. I also need to know how to use it

**Reply by LitoStarr** [reply]
> unfortunately I am inept, but I want to install the DCOF program that you will release. It doesn't matter if I have to pay you, just tell me and we will arrange the price and payment method. I remain at your service. I also need to know how to use it

I can assist you with this.
Discord: litostarr#1928
